lib/new_relic/agent/database.rb

```diff
@@ -42,6 +42,7 @@ def capture_query(query)
       end
 
       def truncate_query(query)
+        return unless query
```

Comment:
I was surprised to see this covered by SimpleCov. We may want to add a test somewhere for this functionality.

Reply:
added here: 693490fbdef49afc063c70586cfb25eb570fdc91